README.md

```diff
@@ -133,13 +133,15 @@ api_prefix = "/api/v1/"
 api_spec_file = "openapi.json"
 debug = true
 auth = true
+is_local_instance = false
```

Comment:
maybe it's better to name it use_jwt, or use_h_rh_identity?

Reply:
or to make a string `auth_type`

Reply:
+1 that's better option

Reply:
must be `auth_type`